sdk/python/Makefile

```diff
@@ -65,6 +65,16 @@ test_auto:: $(TEST_ALL_DEPS)
 
 test_all:: test_fast test_auto test_go
 
+PULUMI_TEST_ORG ?= $(shell pulumi whoami --json | jq ".organizations[0]")
```

Comment:
Out of curiosity, does running these tests also not work without a `PULUMI_ACCESS_TOKEN` set?

Reply:
I actually got a very specific error that said I can't run tests _without_ a `PULUMI_ACCESS_TOKEN` in the environment for these (and the Go tests too)